docs/pages/inboxes/disappearing-messages.md

```diff
@@ -0,0 +1,86 @@
+# Support disappearing messages with XMTP
+
+Disappearing messages are messages that are intended to be visible to users for only a short period of time. After the message expiration time passes, the messages are removed from the UI and deleted from local storage so the messages are no longer accessible to conversation participants.
+
+Disappearing message behavior is enforced by apps, meaning that apps are responsible for removing messages from their UIs and local storage based on conditions set at the conversation level. Conversation participants using apps that support disappearing messages will have a UX that honors the message expiration conditions. 
+
+However, it's important to understand that:
+
+- A conversation participant using an app that doesn't support disappearing messages will not experience disappearing message behavior.
+- Messages aren't deleted from the XMTP network.
+
+Therefore, disappearing messages should be understood as best-effort, app-level privacy that helps avoid leaving an easily accessible record in a messaging UI. However, it's not a guaranteed, system-wide “message self-destruct.”
+
+## **Implement disappearing messages**
+
+Here is a high-level view of the disappearing messages flow:
+
+1. When a 1:1 or group chat conversation is created or updated, a user with appropriate permissions can set disappearing message conditions for messages sent in the conversation. This includes:
+   - `from_ns`: Starting timestamp from which the message lifespan is calculated
+   - `in_ns`: Duration for which the message should remain visible to conversation participants
+2. In a conversation with disappearing message conditions, a sender creates a message. The message abides by disappearing message metadata set for the conversation.
```

Comment:
Just to be sure I understand - while the conditions are set at the conversation level, the message does not include metadata that indicates the conditions -- rather the dev would get that info by looking at the parent conversation, is that right?

Reply:
Yea, that's correct.